static/lang/re-en.json

```diff
@@ -3091,6 +3091,12 @@
                 "UnholyWater": {
                     "Note": "Can damage only creatures with the holy trait."
                 },
+                "Version": {
+                    "Lesser": "Lesser",
+                    "Greater": "Greater",
+                    "Major": "Major",
+                    "Moderate": "Moderate"
+                },
```

Comment:
We're not making generic labels for these adjectives for now, see this [comment thread](https://github.com/foundryvtt/pf2e/pull/16812#discussion_r1793835413)